File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    var answer = 1;
    let former = 0;
    let next = 1;
    for(let i=0; i<n-1; i++){
        answer = (next+former) % 1234567;    
        former = next;
        next = answer;
    } 
    return answer;
}

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 1;
    int former = 0;
    int next = 1;
    for(int i=0; i<n-1; i++){
        answer = (next+former) % 1234567;    
        former = next;
        next = answer;
    }    
    return answer;
}